camp/sachin-Xbox-Controller-Right-Stick-X-direction/src/main/java/frc/robot/commands/MoveMotorCommand.java

```diff
@@ -0,0 +1,74 @@
+// ************************************************************
```

Comment:
Since this is used as a DefaultCommand, it should have "DefaultCommand" in the file name.

Reply:
you still need to address this comment